scss/v2/components/tile.scss

```diff
@@ -83,6 +121,32 @@
     box-shadow: 0 2px 0 0 rgba(65, 64, 66, 0.35);
   }
 
+  &__highlighted-content-image-container {
+    // Remove margin from the image container so that the image can stretch across the full width of the tile.
```

Comment:
Yeah, I played around with a few things here to try and get it to fit the entire `tile`. So the parent container here is the `tile` class, which had `padding-left/right/bottom` set to `16px`. It made sense to me to keep that as it is because it applies to every element within the `tile` component rather than specifying `padding` properties for each of them. So with that in mind, adding the negative margin would offset that padding for the `image-container` only, and allow its `width` property to fill the full tile width

If you think there's a cleaner solution though I'm happy to go with that

Reply:
Ah right I see, we can remove the padding from the tile if required, for 'main-features' I can add an inner container then. But also I can see the reasoning for this now as per comment so will mark as resolved as happy either way ^^